### DISCUSSION TITLE
"Your GitHub Copilot subscription has ended." despite verified GitHub Education Student status

### DISCUSSION THREAD

**Discussion by haokanga** [community/community#145598] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

https://education.github.com/discount_requests/application shows:
<img width="317" alt="image" src="https://github.com/user-attachments/assets/deb0d5a5-068f-415a-9e3d-4e55043edb48">

https://github.com/settings/copilot shows:
<img width="1188" alt="image" src="https://github.com/user-attachments/assets/d2a15f50-ce2e-4554-9592-a404968c61a0">

**Comment by davevad93** [comment, marked as answer]
Hi @haokanga ,

That's strange indeed, check if you can redeem your offer by going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue attaching screenshots as well.

**Reply by haokanga** [reply]
@davevad93 Thank you for your guidance. I confirmed that using the link in the pack offers page works!

<img width="434" alt="image" src="https://github.com/user-attachments/assets/c49ca935-5e5d-423a-a3b3-42ebe3125f9c">

**Reply by davevad93** [reply]
You're welcome @haokanga , glad you solved it! 👍
